From 651301f7c0dc3ea58d2b770baeb8de0f42ceb609 Mon Sep 17 00:00:00 2001
From: Johnathan Corgan <jcorgan@corganenterprises.com>
Date: Sun, 5 Feb 2012 10:27:47 -0800
Subject: [PATCH 22/47] howto: various fixes cmake, cleanup

* Fixed include dir variable, now works when gnuradio-core in custom prefix

* Removed QUIET option, only works for cmake >=2.8.2, Ubuntu LTS has 2.8.0

* Fixed library name for gruel search
---
 .../cmake/Modules/FindGnuradioCore.cmake           |    4 ++--
 .../cmake/Modules/FindGruel.cmake                  |    4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/gr-howto-write-a-block-cmake/cmake/Modules/FindGnuradioCore.cmake b/gr-howto-write-a-block-cmake/cmake/Modules/FindGnuradioCore.cmake
index b45cf95..3773588 100644
--- a/gr-howto-write-a-block-cmake/cmake/Modules/FindGnuradioCore.cmake
+++ b/gr-howto-write-a-block-cmake/cmake/Modules/FindGnuradioCore.cmake
@@ -1,11 +1,11 @@
 INCLUDE(FindPkgConfig)
-PKG_CHECK_MODULES(PC_GNURADIO_CORE QUIET gnuradio-core)
+PKG_CHECK_MODULES(PC_GNURADIO_CORE gnuradio-core)
 
 FIND_PATH(
     GNURADIO_CORE_INCLUDE_DIRS
     NAMES gr_random.h
     HINTS $ENV{GNURADIO_CORE_DIR}/include/gnuradio
-        ${PC_GNURADIO_CORE_INCLUDE_DIR}
+        ${PC_GNURADIO_CORE_INCLUDEDIR}
     PATHS /usr/local/include/gnuradio
           /usr/include/gnuradio
 )
diff --git a/gr-howto-write-a-block-cmake/cmake/Modules/FindGruel.cmake b/gr-howto-write-a-block-cmake/cmake/Modules/FindGruel.cmake
index 56b8b17..58dff70 100644
--- a/gr-howto-write-a-block-cmake/cmake/Modules/FindGruel.cmake
+++ b/gr-howto-write-a-block-cmake/cmake/Modules/FindGruel.cmake
@@ -1,11 +1,11 @@
 INCLUDE(FindPkgConfig)
-PKG_CHECK_MODULES(PC_GRUEL QUIET gnuradio-core)
+PKG_CHECK_MODULES(PC_GRUEL gruel)
 
 FIND_PATH(
     GRUEL_INCLUDE_DIRS
     NAMES gruel/attributes.h
     HINTS $ENV{GRUEL_DIR}/include
-        ${PC_GRUEL_INCLUDE_DIR}
+        ${PC_GRUEL_INCLUDEDIR}
     PATHS /usr/local/include
           /usr/include
 )
-- 
1.7.9.1

